ccm_web/server/src/main.ts

```diff
@@ -60,13 +62,17 @@ async function bootstrap (): Promise<void> {
     const swaggerConfig = new DocumentBuilder()
       .setTitle('Canvas Course Manager')
       .setDescription('CCM application API description and explorer')
-      .addBearerAuth({
-        type: 'http',
-        description: (
-          'The bearer token can be found in the "token" URL parameter' +
-          ' (use a browser tool to view the URL of the frame).'
-        )
-      })
+      .addSecurity(
+        'CSRF-Token', {
+          type: 'apiKey',
+          name: 'CSRF-Token',
+          in: 'header',
+          description: (
+            'POST and PUT requests need to include a CSRF-Token header. ' +
+            'The token can be found in the "csrfToken" URL parameter ' +
+            '(use a browser tool to view the URL of the frame).'
+          )
+        })
```

Comment:
I did a fair amount of research on how to set this up so we don't have to add the token manually, but I hit several deadends. I'm inclined to keep it like this for now and open a separate issue (I will do that anyway and add the links that I found related to this). One possible solution is to use `swagger-react-ui` (since it's on the client side, it can access the token more easily), but we'd need to serve the Swagger document somehow, which is an extra step I haven't thought through.

Reply:
See https://github.com/tl-its-umich-edu/canvas-course-manager-next/issues/167

Reply:
If that makes using the Swagger UI easier, it will be nice to see.